pkg/pdutil/clock.go

```diff
@@ -120,8 +120,8 @@ func (c *clock) CurrentTime() (time.Time, error) {
 // Stop clock.
 func (c *clock) Stop() {
 	c.mu.Lock()
-	defer c.mu.Unlock()
 	c.cancel()
+	c.mu.Unlock()
```

Comment:
Do we need to add a unit test

Reply:
Actually, this will fix that unstable test. So I don't think we need to add a new test.